.drone/drone.yml

```diff
@@ -106,26 +105,19 @@ trigger:
 
 steps:
   - name: test
-    image: rfratto/seego
+    image: grafana/agent-build-image:0.15.1
     volumes:
       - name: docker
         path: /var/run/docker.sock
     commands:
-      - apt-get update &&  apt-get install -y rubygems rpm nsis apt-transport-https ca-certificates curl gnupg lsb-release libbpfcc-dev
-      - gem install --no-document fpm
-      - rm -r /usr/local/go
-      - mkdir -p /usr/local/go/bin
-      - wget -q https://golang.org/dl/go1.18.linux-amd64.tar.gz
-      - tar -C /usr/local -xzf go1.18.linux-amd64.tar.gz
-      - rm go1.18.linux-amd64.tar.gz
-      - export PATH=$PATH:/usr/local/go/bin
-      - make DOCKER_OPTS="" cmd/agent/agent cmd/agentctl/agentctl cmd/agent-operator/agent-operator tools/crow/grafana-agent-crow tools/smoke/grafana-agent-smoke
-      - make DOCKER_OPTS="" K8S_USE_DOCKER_NETWORK=1 DRONE=true BUILD_IN_CONTAINER=false test
+      - make -f Makefile-v2.mk binaries
+      - K8S_USE_DOCKER_NETWORK=1 make -f Makefile-v2.mk test
```

Comment:
can we have a comment why this env is needed? It seems kinda out of thin air now that the command is simpler.